### ISSUE TITLE
[:new:] Add functionality for describing the lambda-service

### ISSUE THREAD

**Issue by kirstein** [Testlio/lambda-tools#35]
**Type:**
:new:

**Description:**
Add a function that describes lambda-service. Function that shows us to what lambdas the available lambda functions are actually tied to.

**Motivation:**
The problem is that we have different kind of lambda functions. 
We have lambdas that are tied to api-gateway and lambdas that are tied to other service events (such as dynamodb streams). 

This is all good, but all of our lambda functions are in the same folder `/lambdas`, which makes it super hard to differentiate between them and see where and how the lambdas are actually used.

The `describe` function would be perfect for that. It would make jumping into a current lambda service a whole lot easier.

**Example usage:**
```bash
$ lambda describe
```

**Comment by henrinormak** [comment]
The approach for this could leverage the already existing cf.json file, it would however need to know which type of resources are interesting (i.e AWS::Lambda::EventSourceMapping and so on). It would also need to know the internal logic of how the Lambda functions are named in the the CF template (CapitalizedCamelCase).

Finally, as API Gateway is not visible in the template, the describer would also need to know how that is put together (so it would need to have access to the template).

In short, the output could be something along the lines of a tree that at the top level has "API Gateway" and any event sources (DynamoDB, CloudWatch etc.), under which we can list the Lambda functions by name.

**Comment by henrinormak** [comment]
Done.
